daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -48,6 +45,8 @@ commandParser =
          , command "init" (info (initCmd <**> helper) idm)
          , command "start" (info (startCmd <**> helper) idm)
          , command "deploy" (info (deployCmd <**> helper) idm)
+         , command "list-parties" (info (listPartiesCmd <**> helper) idm)
+         , command "allocate-party" (info (allocatePartyCmd <**> helper) idm)
```

Comment:
Like I said beloy, I think these would all be better organized as subcommands of "deploy".

Reply:
Yup. Where are the existing examples?

Reply:
I can dig up an old example for you.

Reply:
Here, from the `da` code base:

https://github.com/digital-asset/daml/blob/1fcf2e4442f102c894af06164c5c0872db85e4b0/da-assistant/DA/Sdk/Cli/Command.hs#L107

You can see `templateActionParser` is defined further below as another `subparser ... subcommand` list. Note that `subcommand` is defined in that file as a convenience around `command ... (info ...)` but you don't really need it.